src/Bicep.Core/Bicep.Core.csproj

```diff
@@ -1,19 +1,48 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
-  </PropertyGroup>
+	<PropertyGroup>
+		<TargetFramework>netstandard2.1</TargetFramework>
+	</PropertyGroup>
 
-  <ItemGroup>
-    <PackageReference Include="JetBrains.Annotations" Version="2021.1.0">
-      <PrivateAssets>all</PrivateAssets>
-    </PackageReference>
-    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
-    <PackageReference Include="System.Collections.Immutable" Version="5.0.0" />
-    <PackageReference Include="Azure.Deployments.Expression" Version="1.0.173" />
-    <PackageReference Include="Azure.Deployments.Core" Version="1.0.173" />
-    <PackageReference Include="Azure.Bicep.Types" Version="0.1.112" />
-    <PackageReference Include="Azure.Bicep.Types.Az" Version="0.1.112" />
-  </ItemGroup>
+	<ItemGroup>
+		<None Remove="bicepconfig.json" />
+	</ItemGroup>
 
+	<ItemGroup>
+		<EmbeddedResource Include="Configuration\bicepconfig.json" />
+	</ItemGroup>
+
+	<ItemGroup>
+		<PackageReference Include="JetBrains.Annotations" Version="2021.1.0">
+			<PrivateAssets>all</PrivateAssets>
+		</PackageReference>
+		<PackageReference Include="Microsoft.Extensions.Configuration" Version="5.0.0" />
+		<PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="5.0.0" />
+		<PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="5.0.0" />
+		<PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
+		<PackageReference Include="System.Collections.Immutable" Version="5.0.0" />
+		<PackageReference Include="Azure.Deployments.Expression" Version="1.0.173" />
+		<PackageReference Include="Azure.Deployments.Core" Version="1.0.173" />
+		<PackageReference Include="Azure.Bicep.Types" Version="0.1.112" />
+		<PackageReference Include="Azure.Bicep.Types.Az" Version="0.1.112" />
+	</ItemGroup>
+	<ItemGroup>
+		<InternalsVisibleTo Include="Bicep.Core.UnitTests" />
```

Comment:
>InternalsVisibleTo [](http://example.com/codeflow?start=3&length=18)

Internal is just Public with extra steps. We should get rid of internals that tests depends on or make them public.